tests/pw/feature-map/feature-map.yml

```
@@ -112,6 +113,7 @@
           vendor can permanently delete product [lite]: true
           vendor can view product [lite]: true
           vendor can't add product without required fields [lite]: true
+          vendor can't add product without required fields [lite]: true
```

yamllint

[error] duplication of key "vendor can't add product without required fields [lite]" in mapping

(key-duplicates)